packages/vite-plugin-cloudflare/e2e/helpers.ts

```diff
@@ -169,7 +170,7 @@ function wrap(proc: childProcess.ChildProcess): Process {
 	return wrappedProc;
 }
 
-async function updateVitePluginVersion(projectPath: string) {
+async function updateVitePluginVersionAndAddWrangler(projectPath: string) {
```

Comment:
Strictly not adding wrangler since it does nothing if wrangler is not already there...

How about:


```suggestion
async function updateVitePluginAndWranglerVersion(projectPath: string) {
```